frontend/packages/kubevirt-plugin/src/selectors/pod/selectors.ts

```diff
@@ -88,20 +84,17 @@ export const findVMIPod = (
 export const getVMImporterPods = (
   vm: VMKind,
   pods?: PodKind[],
-  pvcNameLabel = `${CDI_KUBEVIRT_IO}/${STORAGE_IMPORT_PVC_NAME}`,
+  pvcs?: PersistentVolumeClaimKind[],
 ) => {
   if (!pods) {
     return null;
   }
 
-  const datavolumeNames = getDataVolumeTemplates(vm)
-    .map((dataVolumeTemplate) => getName(dataVolumeTemplate))
-    .filter((dataVolumeTemplate) => dataVolumeTemplate);
-
+  const importerPodNames = pvcs.map((pvc) => getPvcImportPodName(pvc));
```

Comment:
```suggestion
  const importerPodNames = (pvcs || []).map((pvc) => getPvcImportPodName(pvc));
```

Reply:
+1

also better to create a lookup here

Reply:
Done!